TESTS/mbed_hal/sleep_manager/main.cpp

```diff
@@ -68,31 +88,30 @@ utest::v1::status_t testcase_setup(const Case *const source, const size_t index_
 {
     // Suspend the RTOS kernel scheduler to prevent interference with duration of sleep.
     osKernelSuspend();
+
 #if DEVICE_LPTICKER
-    ticker_suspend(get_lp_ticker_data());
```

Comment:
The `lp_ticker_wrapper_suspend()` should not be called without calling `ticker_suspend(get_lp_ticker_data())` first.

https://github.com/ARMmbed/mbed-os/blob/9974899a6b1947f0ad4c729a3608b2d172d6eb28/hal/mbed_lp_ticker_wrapper.h#L52-L61

Reply:
That was done following @mprse earlier comment to suspend the wrapper. But that the same time we have the:  "Modify setup/teardown handler’s: remove suspension of lp/us tickers, so they can count as this is required by test_lock_unlock_test_check test case." .. so I guess that we should go back to NOT suspending the lp_ticker_wrapper - @mprse : do you think that would be ok ?